redbot/cogs/downloader/downloader.py

```diff
@@ -1388,20 +1388,22 @@ async def findcog(self, ctx: commands.Context, command_name: str) -> None:
             cog_name = self.cog_name_from_instance(cog)
             installed, cog_installable = await self.is_installed(cog_name)
             if installed:
-                is_installable = True
+                from_git = True
```

Comment:
A minor nit pick but this var is actually unnecessary and the `installed` var could just be used instead of it as it contains exactly the same bool.